signed-playback-lambda/README.md

```diff
@@ -0,0 +1,46 @@
+# Lambda for Signed Playback
+
```

Comment:
I might add a sentence or two explaining that this uses Mux and link to the website- just in case this directory get's linked outside of the examples repo for whatever reason